woocommerce/woocommerce-gutenberg-products-block
# Storybook and TS migration of ReadMore component

This PR adds stories for the `ReadMore` component and migrates it to TypeScript. It also migrates a bunch of associated functions and components.

Fixes #5254

### Testing

### Manual Testing

How to test the changes in this Pull Request:

1. Run `npm run storybook`
2. Explore the [ReadMore stories](http://localhost:6006/?path=/story/woocommerce-blocks-base-components-readmore--default). You should see one default story with the appropriate controls.

> ⚠️  **Please note:** Component will not properly react with control changes. This is because of the way component was written. Fixing this is not part of this PR. See #5299

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).
